README.md

```diff
@@ -1,2 +1,89 @@
-## Loopster
-Сервисная библиотека, которая обеспечивает выполнение сервисных процессов, следит за их зависаниями и при необходимости осуществляет перезапуск.
+# Loopster
+
+Loopster is a service library that provides mechanisms for executing and managing services, including monitoring their status and automatically restarting them when necessary.
+
+## Installation
+
+To install Loopster, you can use pip:
+
+```
+pip install loopster
+```
+
+## Usage
+
+### Basic Service Example
+
+Here's an example of how to create and run a basic service using Loopster:
+
+```python
+from loopster.hubs import base
+from loopster.hubs.controllers import panic
+from loopster.hubs.drivers import process
+from loopster.services import softirq
+
+class BasicService(softirq.SoftIrqService):
+    def _step(self):
+        print("Hello, world!")
+
+driver = process.ProcessDriver()
+controller = panic.PanicController()
+hub = base.BaseHub(driver=driver, controller=controller)
+
+hub.add_service(svc_class=BasicService, svc_kwargs={})
+hub.serve()
+```
+
+### Nested Service Example with MySQLWrapper
+
+Here's an example of how to use a nested service with the `mysqlwrapper` module:
+
+```python
+from restalchemy.storage.sql import engines
+from rooster.hubs.controllers import force_state
+from loopster.hubs.drivers import process
+from loopster.services import mysql
+from loopster.services import softirq
+
+class BasicService(softirq.SoftIrqService):
+    def _step(self):
+        print("Hello, world!")
+
+hub = process.ProcessHub(
+    controller=force_state.AlwaysForceTargetStateController())
+
+hub.add_service(
+    svc_class=mysql.MySQLStorageWrapper,
+    svc_kwargs=dict(
+        nested_class=BasicService,
+        nested_kwargs=dict(
+            step_period=STEP_PERIOD,
+        ),
+        mysql_connection_url=engines.DBConnectionUrl(CONNECTION_URL)
+    )
+)
+```
+
+## Public Interface
+
+### Services
+
+Loopster provides several built-in services, including:
+
+- `SoftIrqService`: A service with a watchdog that runs in an infinite loop.
+- `BjoernService`: A special server for Bjoern, which implements multiprocessing by itself.
+
+### Hub
+
+The `BaseHub` class is used to manage multiple services using one strategy provided by the driver. It inherits from `SoftIrqService`.
```

Comment:
May be listing all hub types?

Reply:
Actually, we have only one now - ProcessHub, described it and added more info.